Sources/Instrumentation/Instrument.swift

```diff
@@ -28,9 +39,13 @@ public protocol Extractor {
 }
 
 /// Conforming types are used to inject values into a specific `Carrier`.
-public protocol Injector {
+public protocol Injector: _SwiftInstrumentationSendable {
     /// The carrier to inject values into.
+    #if swift(>=5.6.0)
+    associatedtype Carrier: Sendable
+    #else
     associatedtype Carrier
+    #endif
```

Comment:
Why has Carrier to be Sendable?

Reply:
I guess it doesn't have to -- I'll remove